src/routes/routes.ts

```diff
@@ -18,82 +19,48 @@ routes.post(
-    res.status(response.statusCode).json(response.body)
-  },
-)
+routes.get('/get_all_subjects', async (req: Request, res: Response) => {
+  const httpRequest: HttpRequest = new HttpRequest(req.body, {}, {}, req.file)
+  const response = await GetAllSubjectsPresenter(httpRequest, repo)
+  res.status(response.statusCode).json(response.body)
+})
 
-routes.put(
-  '/update_availabilities',
-  async (req: Request, res: Response) => {
-    const httpRequest: HttpRequest = new HttpRequest(
-      req.body,
-      {},
-      {},
-      undefined
-    )
-    const response = await UpdateAvailabilitiesPresenter(httpRequest, repo)
-    res.status(response.statusCode).json(response.body)
-  },
-)
+routes.put('/update_availabilities', async (req: Request, res: Response) => {
+  const httpRequest: HttpRequest = new HttpRequest(req.body, {}, {}, undefined)
+  const response = await UpdateAvailabilitiesPresenter(httpRequest, repo)
+  res.status(response.statusCode).json(response.body)
+})
 
-routes.get(
-  '/get_all_professors',
-  async (req: Request, res: Response) => {
-    const httpRequest: HttpRequest = new HttpRequest(
-      req.body,
-      {},
-      {},
-      req.file
-    )
-    const response = await GetAllProfessorsPresenter(httpRequest, repo)
-    res.status(response.statusCode).json(response.body)
-  },
-)
+routes.get('/get_all_professors', async (req: Request, res: Response) => {
+  const httpRequest: HttpRequest = new HttpRequest(req.body, {}, {}, req.file)
+  const response = await GetAllProfessorsPresenter(httpRequest, repo)
+  res.status(response.statusCode).json(response.body)
+})
 
-routes.get(
-  '/get_all_schedules',
-  async (req: Request, res: Response) => {
-    const httpRequest: HttpRequest = new HttpRequest(
-      req.body,
-      {},
-      {},
-      req.file
-    )
-    const response = await GetAllSchedulesPresenter(httpRequest, repo)
-    res.status(response.statusCode).json(response.body)
-  },
-)
+routes.get('/get_all_schedules', async (req: Request, res: Response) => {
+  const httpRequest: HttpRequest = new HttpRequest(req.body, {}, {}, req.file)
+  const response = await GetAllSchedulesPresenter(httpRequest, repo)
+  res.status(response.statusCode).json(response.body)
+})
 
-routes.get(
-  '/get_role_by_email',
-  async (req: Request, res: Response) => {
-    const { email } = req.query; 
+routes.get('/get_role_by_email', async (req: Request, res: Response) => {
+  const { email } = req.query
+
+  const httpRequest: HttpRequest = new HttpRequest({ email }, {}, {}, req.file)
+  const response = await GetRoleByEmailPresenter(httpRequest, repo)
+  res.status(response.statusCode).json(response.body)
+})
 
-    const httpRequest: HttpRequest = new HttpRequest({ email }, {}, {}, req.file);
-    const response = await GetRoleByEmailPresenter(httpRequest, repo);
-    res.status(response.statusCode).json(response.body);
-  }
-);
+routes.get('/get_suitabilities', async (req: Request, res: Response) => {
```

Comment:
`get_suitabilities_by_professor `(front end will wait this, not `get_suitabilities`)